setup.py

```diff
@@ -22,6 +22,9 @@
     "qiskit-terra>=0.16.0",
 ]
 
+jax_extras = ['jax>=0.2.21',
```

Comment:
Do we actually need the latest version of jax, or do the changes you made to fix the module for the latest version of jax still work with older versions?

Reply:
I don't actually know - I believe CI is currently just installing the latest version though, so if we want to list older versions we should add them to the tests. What are your thoughts? I feel like there isn't much need at this stage, especially since it's the first release.